docroot/modules/custom/mass_content/includes/mass_content.org_migration.inc

```diff
@@ -189,7 +189,15 @@ function _mass_content_org_page_migration_news(&$node) {
  * Migrate data for the board members section.
  */
 function _mass_content_org_page_migration_board(&$node) {
-
+  if (!empty($node->field_boards->target_id)) {
+    $pid = $node->field_boards->target_id;
+    $pid_revision = $node->field_boards->target_revision_id;
+    $node->set("field_boards", []);
```

Comment:
The field_boards paragraph needs to be referenced within a new organization section, right?

Reply:
Yes